2023-04-17 09:28:22,795 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-17 09:30:33,708 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-17 09:56:55,421 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-17 15:34:39,123 django.request WARNING  Not Found: /
2023-04-17 15:34:39,263 django.request WARNING  Not Found: /favicon.ico
2023-04-17 16:00:01,574 django.request WARNING  Bad Request: /api/auth/users/
2023-04-17 16:00:30,312 django.request WARNING  Bad Request: /api/auth/users/
2023-04-17 16:06:33,983 django.request ERROR    Internal Server Error: /api/auth/users/me
Traceback (most recent call last):
  File "D:\Projects\helpdesk\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\helpdesk\.venv\Lib\site-packages\django\utils\deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\helpdesk\.venv\Lib\site-packages\django\middleware\common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\helpdesk\.venv\Lib\site-packages\django\middleware\common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 127.0.0.1:8000/api/auth/users/me/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
2023-04-17 16:14:47,900 django.request WARNING  Unauthorized: /api/auth/users/set_password/
2023-04-17 16:14:56,352 django.request WARNING  Unauthorized: /api/auth/users/set_password/
2023-04-17 16:15:05,761 django.request WARNING  Unauthorized: /api/auth/users/set_password/
2023-04-17 16:16:21,516 django.request WARNING  Not Found: /api/auth//users/reset_password/
2023-04-17 16:16:29,997 django.request WARNING  Not Found: /api/auth//users/reset_password/
2023-04-17 16:22:00,605 django.utils.autoreload INFO     D:\Projects\helpdesk\backend\core\settings.py changed, reloading.
2023-04-17 16:22:11,661 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-17 16:22:23,766 django.utils.autoreload INFO     D:\Projects\helpdesk\backend\core\settings.py changed, reloading.
2023-04-17 16:22:27,013 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-17 16:22:52,635 django.utils.autoreload INFO     D:\Projects\helpdesk\backend\core\settings.py changed, reloading.
2023-04-17 16:22:56,443 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-17 17:35:51,354 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-17 17:36:34,344 django.request WARNING  Unauthorized: /api/auth/users/reset_password_confirm/
2023-04-17 17:36:35,406 django.request WARNING  Not Found: /favicon.ico
2023-04-18 15:22:15,495 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-18 15:22:57,231 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 12:09:01,771 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 12:26:00,455 django.utils.autoreload INFO     D:\Projects\basic\backend\core\urls.py changed, reloading.
2023-04-22 12:26:05,623 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 12:27:42,396 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\serializers.py changed, reloading.
2023-04-22 12:27:44,252 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 12:30:58,629 django.request WARNING  Not Found: /favicon.ico
2023-04-22 13:43:06,743 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 13:43:08,724 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 14:07:30,336 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 14:07:32,479 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 14:07:48,507 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 14:07:50,258 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 14:12:05,655 django.request WARNING  Not Found: /ticket/api/ticket_details/1
2023-04-22 14:12:06,031 django.request WARNING  Not Found: /ticket/api/ticket_details/1
2023-04-22 14:12:10,467 django.request WARNING  Not Found: /ticket/api/ticket_details/1
2023-04-22 14:12:15,710 django.request WARNING  Not Found: /ticket/api/ticket_details/1
2023-04-22 14:15:05,201 django.request WARNING  Method Not Allowed: /api/token/
2023-04-22 14:17:28,021 django.request WARNING  Not Found: /ticket/api/ticket_details/1
2023-04-22 14:19:43,769 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 14:19:45,887 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 17:21:41,881 django.utils.autoreload INFO     D:\Projects\basic\backend\core\settings.py changed, reloading.
2023-04-22 17:21:46,648 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 17:23:57,281 django.request WARNING  Bad Request: /api/auth/users/reset_password/
2023-04-22 17:24:22,665 django.request WARNING  Unauthorized: /api/token/
2023-04-22 17:24:45,927 django.request WARNING  Unauthorized: /api/token/
2023-04-22 17:27:33,295 django.request WARNING  Bad Request: /api/auth/users/reset_password/
2023-04-22 17:30:45,841 django.request WARNING  Bad Request: /api/auth/users/reset_password/
2023-04-22 17:33:16,851 django.request WARNING  Unauthorized: /api/auth/users/reset_password/
2023-04-22 19:54:59,032 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 20:09:59,556 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:10:40,698 django.request WARNING  Unauthorized: /api/token/
2023-04-22 20:11:12,560 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:13:19,888 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:15:39,524 django.request WARNING  Unauthorized: /api/token/
2023-04-22 20:16:29,187 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:16:40,363 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:17:53,262 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:19:51,993 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:20:16,571 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:20:49,141 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:23:11,880 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:23:27,637 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:26:47,074 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:31:52,650 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:36:40,882 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:46:01,264 django.request WARNING  Unauthorized: /api/token/
2023-04-22 20:48:55,515 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 20:51:20,158 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 20:52:54,297 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-22 21:18:39,289 django.request WARNING  Unauthorized: /api/token/
2023-04-22 21:31:11,516 django.request WARNING  Unauthorized: /api/token/
2023-04-22 21:33:35,730 django.request WARNING  Unauthorized: /api/token/
2023-04-22 21:33:50,528 django.request WARNING  Unauthorized: /api/token/
2023-04-22 21:33:58,542 django.request WARNING  Unauthorized: /api/token/
2023-04-22 21:49:52,151 django.request WARNING  Unauthorized: /ticket/api/create-ticket/
2023-04-22 21:51:27,716 django.request ERROR    Internal Server Error: /ticket/api/create-ticket/
Traceback (most recent call last):
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\backend\ticket\views.py", line 157, in api_create_ticket
    data.created_by=request.user
    ^^^^^^^^^^^^^^^
AttributeError: 'dict' object has no attribute 'created_by'
2023-04-22 21:53:07,039 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 21:53:10,851 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 21:53:22,908 django.request WARNING  Bad Request: /ticket/api/create-ticket/
2023-04-22 21:56:43,924 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 21:56:45,717 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 21:56:50,865 django.request WARNING  Bad Request: /ticket/api/create-ticket/
2023-04-22 21:56:53,566 django.request WARNING  Bad Request: /ticket/api/create-ticket/
2023-04-22 21:57:51,514 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 21:57:53,299 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 21:57:58,088 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 21:57:59,822 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 21:58:05,427 django.request ERROR    Internal Server Error: /ticket/api/create-ticket/
Traceback (most recent call last):
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\backend\ticket\views.py", line 160, in api_create_ticket
    if serializer.is_valid():
       ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\serializers.py", line 227, in is_valid
    self._validated_data = self.run_validation(self.initial_data)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\serializers.py", line 426, in run_validation
    value = self.to_internal_value(data)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\serializers.py", line 483, in to_internal_value
    validated_value = field.run_validation(primitive_value)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\relations.py", line 153, in run_validation
    return super().run_validation(data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\fields.py", line 547, in run_validation
    value = self.to_internal_value(data)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\relations.py", line 264, in to_internal_value
    return queryset.get(pk=data)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\db\models\query.py", line 633, in get
    num = len(clone)
          ^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\db\models\query.py", line 380, in __len__
    self._fetch_all()
  File "D:\Projects\basic\.venv\Lib\site-packages\django\db\models\query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1560, in execute_sql
    cursor.execute(sql, params)
  File "D:\Projects\basic\.venv\Lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OverflowError: Python int too large to convert to SQLite INTEGER
2023-04-22 21:58:28,166 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 21:58:30,074 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 21:59:38,017 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 21:59:39,918 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 21:59:49,582 django.request WARNING  Bad Request: /ticket/api/create-ticket/
2023-04-22 21:59:55,811 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 21:59:57,473 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 22:00:07,065 django.request WARNING  Bad Request: /ticket/api/create-ticket/
2023-04-22 22:01:32,090 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 22:01:33,834 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-22 22:01:45,338 django.request WARNING  Bad Request: /ticket/api/create-ticket/
2023-04-22 22:02:41,169 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-22 22:02:42,963 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 06:33:47,827 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 06:35:07,314 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-23 06:35:14,645 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-23 06:35:35,120 django.request WARNING  Unauthorized: /api/token/
2023-04-23 06:41:47,132 django.request WARNING  Not Found: /ticket/api/ticket-details/[object Object]
2023-04-23 06:41:52,939 django.request WARNING  Not Found: /ticket/api/ticket-details/[object Object]
2023-04-23 06:45:05,904 django.request WARNING  Not Found: /ticket/api/ticket-details/[object Object]
2023-04-23 10:38:31,293 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-23 10:38:54,930 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-23 10:42:16,665 django.request WARNING  Unauthorized: /ticket/api/all-tickets/
2023-04-23 11:10:17,737 django.request WARNING  Not Found: /ticket/api/update-details/1
2023-04-23 11:10:21,137 django.request WARNING  Not Found: /ticket/api/update-details/1
2023-04-23 11:10:41,843 django.request WARNING  Not Found: /ticket/api/update-details/1
2023-04-23 11:10:53,812 django.request WARNING  Not Found: /ticket/api/update-details/1
2023-04-23 11:11:51,765 django.request WARNING  Not Found: /ticket/api/update-ticket/1
2023-04-23 11:11:59,248 django.request WARNING  Not Found: /ticket/api/update-ticket/1
2023-04-23 11:14:52,806 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\urls.py changed, reloading.
2023-04-23 11:15:01,684 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 11:15:21,808 django.request WARNING  Method Not Allowed: /ticket/api/update-ticket/1/
2023-04-23 11:20:50,745 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 11:20:52,832 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 11:24:07,628 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 11:24:09,501 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 11:24:12,474 django.request ERROR    Internal Server Error: /ticket/api/update-ticket/1/
Traceback (most recent call last):
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\request.py", line 416, in __getattr__
    return getattr(self._request, attr)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'WSGIRequest' object has no attribute 'Method'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\views\decorators\csrf.py", line 56, in wrapper_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 480, in raise_uncaught_exception
    raise exc
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\decorators.py", line 50, in handler
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\backend\ticket\views.py", line 171, in api_update_ticket
    if request.Method=="GET":
       ^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\request.py", line 418, in __getattr__
    return self.__getattribute__(attr)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Request' object has no attribute 'Method'
2023-04-23 11:25:00,146 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 11:25:01,928 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 11:42:03,442 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 11:42:05,216 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 11:42:31,389 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 11:42:33,323 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 11:47:55,652 django.request WARNING  Bad Request: /ticket/api/update-ticket/3/
2023-04-23 11:48:05,237 django.request WARNING  Bad Request: /ticket/api/update-ticket/3/
2023-04-23 11:48:14,414 django.request WARNING  Bad Request: /ticket/api/update-ticket/3/
2023-04-23 11:48:32,554 django.request WARNING  Bad Request: /ticket/api/update-ticket/3/
2023-04-23 11:48:44,075 django.request WARNING  Bad Request: /ticket/api/update-ticket/3/
2023-04-23 11:50:48,074 django.request WARNING  Bad Request: /ticket/api/update-ticket/3/
2023-04-23 11:51:47,448 django.request WARNING  Bad Request: /ticket/api/update-ticket/3/
2023-04-23 11:52:55,614 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 11:52:57,459 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 11:53:54,919 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 11:53:57,116 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 11:54:03,598 django.request WARNING  Bad Request: /ticket/api/update-ticket/3/
2023-04-23 11:58:54,822 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 11:58:56,586 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 11:59:05,895 django.request ERROR    Internal Server Error: /ticket/api/update-ticket/3/
Traceback (most recent call last):
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\template\response.py", line 111, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\renderers.py", line 99, in render
    ret = json.dumps(
          ^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\utils\json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ShishirGupta\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\ShishirGupta\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ShishirGupta\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
TypeError: keys must be str, int, float, bool or None, not module
2023-04-23 12:01:12,253 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 12:01:13,894 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 12:01:40,491 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 12:01:42,213 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 12:01:49,294 django.request ERROR    Internal Server Error: /ticket/api/update-ticket/3/
Traceback (most recent call last):
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\template\response.py", line 111, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\renderers.py", line 99, in render
    ret = json.dumps(
          ^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\utils\json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ShishirGupta\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\ShishirGupta\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ShishirGupta\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
TypeError: keys must be str, int, float, bool or None, not module
2023-04-23 12:02:49,505 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 12:02:51,351 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 12:02:56,916 django.request ERROR    Internal Server Error: /ticket/api/update-ticket/3/
Traceback (most recent call last):
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
               ^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\django\template\response.py", line 111, in render
    self.content = self.rendered_content
                   ^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\renderers.py", line 99, in render
    ret = json.dumps(
          ^^^^^^^^^^^
  File "D:\Projects\basic\.venv\Lib\site-packages\rest_framework\utils\json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ShishirGupta\AppData\Local\Programs\Python\Python311\Lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
          ^^^^^^^^^^^
  File "C:\Users\ShishirGupta\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 200, in encode
    chunks = self.iterencode(o, _one_shot=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ShishirGupta\AppData\Local\Programs\Python\Python311\Lib\json\encoder.py", line 258, in iterencode
    return _iterencode(o, 0)
           ^^^^^^^^^^^^^^^^^
TypeError: keys must be str, int, float, bool or None, not module
2023-04-23 12:04:08,293 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 12:04:10,190 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 12:04:41,066 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 12:04:43,136 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 12:04:58,135 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 12:05:00,073 django.utils.autoreload INFO     Watching for file changes with StatReloader
2023-04-23 12:06:15,418 django.utils.autoreload INFO     D:\Projects\basic\backend\ticket\views.py changed, reloading.
2023-04-23 12:06:17,637 django.utils.autoreload INFO     Watching for file changes with StatReloader
